stand-alone/standalone-loader.tsx

```diff
@@ -80,6 +80,7 @@ window.loadPlayer = async (context: RedVenturePlayerContextProps) => {
 			jwPlayerContainerEmbedId: getJwPlayerContainerEmbedId(context),
 			playerUrl: context?.playerUrl,
 			autoPlay: context?.autoPlay,
+			hasAds: context.hasAds,
```

Comment:
Based on `RedVenturePlayerContext` interface, this parameter is optional and could potentially be undefined.

```suggestion
			hasAds: context?.hasAds,
```

Reply:
The `hasAds` is optional but `context` is not. I think these `?` above are redundant.

Reply:
Thanks for the clarification. You could be right, sir. However, something still seems a bit off to me 🤔 I might have misunderstood the changes, but shouldn't its value default to true? If it's undefined, that wouldn't happen, would it? 😬

Reply:
Discussed directly 👍🏻